capabilityquerier/pkg/capabilityquerier/capabilityquerier.go

```diff
@@ -104,7 +121,6 @@ func requestCapabilityStatement(ctx context.Context, fhirURL *url.URL, client *h
 		}
 	}
 
-	message.HTTPResponse = httpResponseCode
 	message.TLSVersion = tlsVersion
```

Comment:
does the TLSVersion, MimeType, etc. get overwritten depending on the order this is called?
I feel like we should either:
* write these once for the `metadata` call only
* capture TLS/MimeType/etc. explicitly for both the `metadata` and `well-known` calls.

Additionally, there's a fair amount of logic in here for calling the endpoint multiple times if a particular mime type doesn't work as expected. Again - we should either do that intentionally by capturing the different mime types for each endpoint type we query, or only do those duplicate calls once for the `metadata` endpoint.

I don't have a strong preference which we do... capturing for everything might be easiest because you just add filling out all of those fields into the switch statements you already have and then we have that data for both endpoints. However, we probably care the most about having that info for the metadata endpoints.

Reply:
Modified this so that the TLS and MimeType are only being captured for metadata endpoint